spec/main_spec.rb

```diff
@@ -0,0 +1,184 @@
+# spec :main_spec.rb
+require './main.rb'
+
+RSpec.describe Enumerable do
+  let(:array) { [11, 2, 3, 56] }
+
```

Comment:
You should test other types of enumerables: ranges, array of strings, a block of code.